PublicationsRouter/v2/oai-pmh/README.md

```diff
@@ -0,0 +1,28 @@
+# PubRouter OAI-PMH endpoint
+
+This document describes how to interact with the PubRouter OAI-PMH endpoint to retrieve the metadata records of successfully
+routed notifications.
+
+You may also wish to read the [OAI-PMH documentation](http://www.openarchives.org/OAI/openarchivesprotocol.html).
+
+This endpoint only supports oai_dc formatted metadata, and can only access notifications going back 3 months.
+
+## Routed notifications for a specific repository
+
+The base endpoint of all OAI-PMH requests to PubRouter is: 
+
+    https://pubrouter.jisc.ac.uk/oaipmh/repo/<repo id>
+
+Where "repo_id" is the account id of the repository of whose notifications to retrieve.

```

Comment:
`Where "institution_id" is the account id of the Institution whose notifications are to be retrieved.`